src/Uno.UI/Microsoft/UI/Xaml/Controls/TreeView/TreeView.cs

```diff
@@ -0,0 +1,358 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public partial class TreeView : Control
+	{
+		private const string c_listControlName = "ListControl";
+
+		private TreeViewNode m_rootNode;
+		private IList<TreeViewNode> m_pendingSelectedNodes;
+
+		public TreeView()
+		{
+			DefaultStyleKey = typeof(TreeView);
+
+			m_rootNode = new TreeViewNode();
+			m_pendingSelectedNodes = new List<TreeViewNode>();
+
+			//this.RegisterDisposablePropertyChangedCallback((s, p, e) => OnPropertyChanged(e));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)